timemachine/cpp/src/local_md_utils.hpp

```diff
@@ -0,0 +1,138 @@
+#include "langevin_integrator.hpp"
+#include "nonbonded_all_pairs.hpp"
+#include "nonbonded_interaction_group.hpp"
+
+#include "fanout_summed_potential.hpp"
+#include "summed_potential.hpp"
+
+#include <typeinfo>
+
+namespace timemachine {
+
+bool is_nonbonded_all_pairs_potential(std::shared_ptr<Potential> pot) {
```

Comment:
Should the implementations be in a `.cpp` or `.cu` file? (seems like we'd run into multiple definitions issues if `local_md_utils.hpp` is included in multiple places)

Reply:
Fixed in [24a5d72](https://github.com/proteneer/timemachine/pull/975/commits/24a5d72de80e2e0b3814f90b005ad00aed907a05)

Along with moving a function that was defined here that was in the wrong place